README.md

```diff
@@ -187,6 +187,8 @@ Both `--exclude` and `--include` options can be used together but `--exclude` ta
 
 In this mode, Coverlet doesn't require any additional setup other than including the NuGet package in the unit test project. It integrates with the `dotnet test` infrastructure built into the .NET Core CLI and when enabled, will automatically generate coverage results after tests are run.
 
+If a parameter takes multiple comma-separated values please note that [you will have to add escaped quotes around the string](https://github.com/Microsoft/msbuild/issues/2999#issuecomment-366078677) like this: `/p:Exclude=\"[coverlet.*]*,[*]Coverlet.Core*\"` or `/p:Include=\"[coverlet.*]*,[*]Coverlet.Core*\"`.
```

Comment:
`parameter` => `property`. Also add an example for specifying multiple formats

Reply:
done 👍